.eslintignore

```diff
@@ -1,12 +1,10 @@
-bin/**
-build/**
-conf/**
-coverage/**
-docs/**
-scripts/**
-templates/**
-tests/bench/**
-tests/fixtures/**
-tests/performance/**
-tmp/**
-test.js
```

Comment:
I would add `test.js` back as it is also ignored in `.gitignore`. The benefits of this file as its easy to do ad hoc eslint runs for different issue when you are working on eslint project and never worry about the file to get checked in or linted.